DEVELOPMENT.md

```diff
@@ -115,3 +139,49 @@ bazel build third_party/chromium
 
 # Lastly, put the address returnd by the web server into the Chromium.
 ```
+
+#### Running webapp tests
+
+tl;dr. Use `ibazel test` for regular work (supports limiting the test amount with
+`fit/fdescript` and `console.log` usage) and `ibazel run` for karma console breakpoint
+debugging.
+
+We use karma testing chromium target for our UI unit tests in webapp. However,
+depending on the environments (OSS or internal TB.corp), the commands we use and
+the functionalities are slightly different.
+
+1.  Just run all webapp tests. The job stops after finished. `console.log` is not
+    supported. Not handy on development.
+
+    ```shell
+    (tf)$ bazel test //tensorboard/webapp:karma_test_chromium-local
```

Comment:
There are some webapp tests now that are not captured by this target. To make sure you get them all:

```
(tf)$ bazel test //tensorboard/webapp/...
```

At the moment this runs the following tests:
```
//tensorboard/webapp:karma_test_chromium-local                           PASSED in 42.3s
//tensorboard/webapp/feature_flag:karma_test_chromium-local              PASSED in 5.8s
//tensorboard/webapp/hparams:karma_test_chromium-local                   PASSED in 5.7s
//tensorboard/webapp/notification_center:notification_center_test_chromium-local PASSED in 10.1s
//tensorboard/webapp/persistent_settings:karma_test_chromium-local       PASSED in 10.9s
//tensorboard/webapp/settings:karma_test_chromium-local                  PASSED in 6.9s

```